### PR TITLE
Add PinotDB Doc

### LINKED ISSUE #10030: Add Docs: PinotDB & SQLLite
add docs related to pinotdb & Sqllite database source.

### PR DESCRIPTION
**Pull request by MilanBariya**
### Describe your changes:

Fixes <issue-number>

I worked on adding PinotDB doc
fix #10030

#
### Type of change:

- [ ] Bug fix
- [ ] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

**Review by NiharDoshi99 — APPROVED** [review]

File: openmetadata-docs-v1/content/v0.13.2/connectors/database/pinotdb/airflow.md [hunk 1]
```diff
@@ -0,0 +1,736 @@
...
+
+| Feature            | Status                       |
+| :----------------- | :--------------------------- |
+| Stage              | PROD                         |
+| Metadata           | {% icon iconName="check" /%} |
+| Query Usage        | {% icon iconName="cross" /%} |
+| Data Profiler      | {% icon iconName="check" /%} |
+| Data Quality       | {% icon iconName="check" /%} |
+| Lineage            | Partially via Views          |
+| DBT                | {% icon iconName="cross" /%} |
```
Comment (OnkarVO7) [inline openmetadata-docs-v1/content/v0.13.2/connectors/database/pinotdb/airflow.md:18]: @MilanBariya please add the dbt as `check`

File: openmetadata-docs-v1/content/v1.0.0/menu.md [hunk 2]
```diff
@@ -303,6 +303,12 @@ site_menu:
     url: /connectors/database/mysql/airflow
   - category: Connectors / Database / MySQL / CLI
     url: /connectors/database/mysql/cli
+  - category: Connectors / Database / PinotDB
```
Comment (OnkarVO7) [inline openmetadata-docs-v1/content/v1.0.0/menu.md:306]: Can we arrange this alphabetically

**Review by OnkarVO7 — CHANGES_REQUESTED** [review]
Left a few comments

File: openmetadata-docs-v1/content/v0.13.2/connectors/database/pinotdb/airflow.md (hunk 1 above)
Comment (MilanBariya) [reply openmetadata-docs-v1/content/v0.13.2/connectors/database/pinotdb/airflow.md]: Done

File: openmetadata-docs-v1/content/v1.0.0/menu.md (hunk 2 above)
Comment (MilanBariya) [reply openmetadata-docs-v1/content/v1.0.0/menu.md]: Done

**Review by OnkarVO7 — APPROVED** [review]

**Pull request merged**